### PR TITLE
sc-15765: fix npm package distribution

### PR DESCRIPTION
**Pull request by matthewparavati**
Changes the ios/Frameworks/ folder structure

### CODE REVIEW

**Review by oliveirasWell — APPROVED** [review]

**Comment by oliveirasWell** [comment]
Will you change the version to `1.3.2` in advance?

**Comment by matthewparavati** [comment]
> Will you change the version to `1.3.2` on advance?

Yeah, when you run `npm version` to update the version, it automatically creates a new commit. I'll probably wait until this is merged in to run it and publish a new version

**Pull request merged**
